### ISSUE TITLE
[en][mage][spell/talents] Cone of Cold and Brain freeze / fingers of frost

### ISSUE THREAD

**Issue by blinkmaster** [Atlantiss/BugTracker#6285]
http://www.wowhead.com/spell=120/cone-of-cold
http://www.wowhead.com/spell=44549/brain-freeze
http://www.wowhead.com/spell=112965/fingers-of-frost

how it should work:
hitting 1 target with CoC with brain freeze and fingers of frost talent should have possibility to:
- grant you 1 fof stack ( works properly)
- grant you brain freeze ( doesn't proc BF at all)
- grant you impact proc ( works properly )
hitting 2 and more targets with CoC have possibility to : 
- proc 1+ stacks of fof ( doesn't work at all - > probably coc calculates chill effect hits only on 1 target )
- proc brain freeze separately from all targets ( dunno if it works due to fact brain freeze isn't procced at all by cone of cold)
- grants you impact proc with chance on every hit target ( works properly)

tl;dr : 
CoC doesn't proc multiple FoFs when hits more than 1 target
CoC doesn't proc Brain Freeze at all

video proof of multiple FoF proccs and brain freeze proc:
https://www.youtube.com/watch?v=lihXxekzHdM
1:14 - 1:17 coc usage double FOFs
0:43 - > 0:45 coc brain freeze proc

**Comment by Kejmil** [comment]
@Aviannaqt fixed in rev 4889
